src/goto-symex/goto_symex_state.h

```diff
@@ -115,6 +156,10 @@ class goto_symex_statet final
   // do dereferencing
   value_sett value_set;
 
+  /// Container for data that varies per program point, e.g. the constant
+  /// propagator state, when state needs to branch. This is copied out of
+  /// goto_symex_statet at a control-flow fork and then back into it (at some
```

Comment:
back into it at a control-flow merge

Strike "and not the other way around," since we've just said state goes into it *and* out of it